### PR TITLE
Replace config prefixes 'jdbc-' with 'pg-'

### PR DESCRIPTION
**Pull request by mkubala**

### CODE REVIEW

File: docs/src/main/paradox/index.md [hunk 1]
```diff
@@ -45,8 +45,8 @@ The plugin relies on @extref[Slick](slick:) to do create the SQL dialect for the
 
 Configure `akka-persistence`:
```
Comment (devcateu) [inline docs/src/main/paradox/index.md:46]: shouldnt we change build.sbt also?

File: core/src/main/resources/reference.conf [hunk 2]
```diff
@@ -21,7 +21,7 @@ akka-persistence-jdbc {
   # This is necessary because we must keep track of the highest sequence number that was ever used on a
```
Comment (devcateu) [inline core/src/main/resources/reference.conf:21]: akka-persistence-jdbc - to akka-persistence-pg ?

Comment (devcateu) [reply core/src/main/resources/reference.conf:21]: or akka-persistence-postgres

File: docs/src/main/paradox/index.md (hunk 1 above)
Comment (mkubala) [reply docs/src/main/paradox/index.md:46]: This will be included in rebranding PR

File: core/src/main/resources/reference.conf (hunk 2 above)
Comment (mkubala) [reply core/src/main/resources/reference.conf:21]: I'll change it to akka-persistence-postgres, according to the new plugin name. But this is going to be part of a rebranding story.

**Pull request merged**